frontend/packages/dev-console/src/components/import/import-submit-utils.ts

```diff
@@ -468,6 +470,26 @@ export const createOrUpdateResources = async (
         generatedImageStreamName,
       ),
     );
+  } else if (!_.isEmpty(appResources?.pipeline?.data) && !dryRun) {
```

Comment:
Updated.

Reply:
You combined the two if statements and changed the logic. 

If we have a pipeline template, we don't want a build config. But if it's a dry run, we don't want to do Pipeline stuff because the K8s resources that support DryRun can fail. 

If it's a dry run and we have a template -- you'll request creating a BuildConfig... it's no real harm in the end, but it could theoretically fail the flow and it's just a waste of a call.

```
  if (pipeline.enabled) {
    if (!dryRun) {
      await managePipelineResources(formData, appResources);
    }
  } else {
```
and
```
  if (pipeline.enabled && !dryRun) {
    managePipelineResources(formData, appResources);
  }
  } else {
```
Are two different logic flows. 

Also, always `await` on async methods... rarely do you want to unbound a promise and let it run on it's own without context of this thread -- even if you don't want the response. If an issue happens, it can be confusing where the issue lies if you're already navigating to the Topology screen.

Reply:
Sorry, didn't think it through before combining the statements. Updated.